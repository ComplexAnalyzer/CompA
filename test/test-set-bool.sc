def main() {
    a = [true, true, true];
    print("Len: ");
    print(#a);
    print("false in a");
    print(false?a);
    print("a:");
    (0<=i<#a) print(a[i]);
    a[1] = false;
    print("new a: ");
    (0<=i<#a) print(a[i]);
    print("false in a");
    print(false?a);
    a = set(a);
    print("new a: ");
    (0<=i<#a) print(a[i]);
    print(#a);
}
