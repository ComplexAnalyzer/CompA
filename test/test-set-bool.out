Len: 
3
false in a
false
a:
true
true
true
new a: 
true
false
true
false in a
true
new a: 
true
false
2
